README.md

```diff
@@ -393,19 +393,19 @@ Example:
 Here are examples of commands you can use to upgrade the "default" and "visibility" schemas in your "bring your own" Cassandra database.
 
 ```
-temporal_v1.0.1 $ temporal-cassandra-tool --tls --tls-ca-file ... --user cassandra-user --password cassandra-password --endpoint cassandra.example.com --keyspace temporal --timeout 120 update --schema-dir ./schema/cassandra/temporal/versioned
+temporal_v1.1.0 $ temporal-cassandra-tool --tls --tls-ca-file ... --user cassandra-user --password cassandra-password --endpoint cassandra.example.com --keyspace temporal --timeout 120 update --schema-dir ./schema/cassandra/temporal/versioned
 
-temporal_v1.0.1 $ temporal-cassandra-tool --tls --tls-ca-file ... --user cassandra-user --password cassandra-password --endpoint cassandra.example.com --keyspace temporal_visibility --timeout 120  update --schema-dir ./schema/cassandra/visibility/versioned
+temporal_v1.1.0 $ temporal-cassandra-tool --tls --tls-ca-file ... --user cassandra-user --password cassandra-password --endpoint cassandra.example.com --keyspace temporal_visibility --timeout 120  update --schema-dir ./schema/cassandra/visibility/versioned
 ```
```

Comment:
Might consider separating these example commands and labeling them. e.g:

Upgrade default schema:
```
<sample cmd>
```

Upgrade visibility schema:
```
<sample cmd>
```

Reply:
ah, good call, thank you, updated.